### DISCUSSION TITLE
newer forge?

### DISCUSSION THREAD

**Discussion by x-j0nnay-x** [Sinytra/ForgifiedFabricAPI#28] [Q&A] [answered]
any chance you guys could get it to work with latest forge? I don't remember a lot from my moding days but i dont think forge changes that much on same minecraft version.

**Comment by Su5eD** [comment, marked as answer]
Updating to later Forge versions on mc 1.20.1 is not planned, as explained in #15

**Reply by matthewperiut** [reply]
will this be updating to neoforge for 1.21 or will it update to forge 1.21
